.expeditor/scripts/generate_verify_pipeline.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Determine the files changed between this PR and master.
+# Group them by plan and use that as the unit of work in 
+# downstream steps.
+plans_changed() {
+  git diff --name-only "${BUILDKITE_BRANCH}" "$(git merge-base ${BUILDKITE_BRANCH} master)" \
+  | cut -f1 -d/ \
+  | sort \
+  | uniq
+}
+
+
+create_linux_verify_pipeline_for() {
+  local plan
+  plan=$1
+
+  echo "--- :habitat: :linux: [$plan] Creating Verify Pipeline"
+  sed "|@@plan@@|$plan|" .expeditor/verify_linux_pipeline.yml.template > .buildkite/verify_linux_pipeline.yml
+  buildkite-agent pipeline upload .expeditor/verify_linux_pipeline.yml
```

Comment:
This bit might deserve a comment to explain it - if I understand this correctly:
 * we upload config to define CI for one plan to buildkite
 * we are re-using the filename `.buildkite/verify_linux_pipeline.yml` for each plan, but that doesn't matter (we are not overwriting config repeatedly) because buildkite only uses that filename for the upload step.
 * The uploaded config only applies to *this instance* of buildkite. If someone opens another PR, they do not get this config. If I open and close my PR and restart buildkite, the new buildkite build also does not have this config.

Reply:
👍   This specifics of how we create the dynamic bits of the pipeline are  _probably_ going to change,  so I haven't written the comments yet. 💯 on the things it should cover though.